api_docs/changelog.md

```diff
@@ -1348,9 +1351,9 @@ No changes; feature level used for Zulip 3.0 release.
 
 **Feature level 19**
 
-* [`GET /events`](/api/get-events): `subscriptions` event with
-  `op="peer_add"` and `op="peer_remove"` now identify the modified
-  stream by a `stream_id` field, replacing the old `name` field.
+* [`GET /events`](/api/get-events): The `subscription` event for
+  `peer_add` and `peer_remove` now identify the modified
```

Comment:
nit: "events", or perhaps "identifies"

Reply:
Went with "events" as that mirrors the other feature level entry.